### ISSUE TITLE
cant connect to hashicorp.com on 443

### ISSUE THREAD

**Issue by EnergeticPixels** [bobthecow/vagrant-mongobox#7]
I am a newbie when it comes to vagrant/virtualbox.  Working on a nodejs/restifyjs project and coming up with development processes for the office.  I am the first one to get into NodeJS (targeting xAPI) at the office.  So setting ground rules for development is paramount.  I have thought that integrating Vagrant would be a easy way to keep everyone on the same page.  I needed a MongoDB instance on a Linux machine.  Found your Vagrant-mongoDB on GH.  But the UP command is havin issues with finding precise64.  It tried from two places.  Can you point me in right direction?

```
PS D:\projects\ep\mongobox> vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'precise64' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
==> default: Box file was not detected as metadata. Adding it directly...
==> default: Adding box 'precise64' (v0) for provider: virtualbox
    default: Downloading: http://files.vagrantup.com/precise64.box
    default: Download redirected to host: hashicorp-files.hashicorp.com
    default:
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

Failed to connect to hashicorp-files.hashicorp.com port 443: Timed out
PS D:\projects\ep\mongobox>
```

**Comment by EnergeticPixels** [comment]
I found the answer after some deep digging.  seems like there was a transfer of hostings/control between hashicorp and vagrant.

see:  https://github.com/hashicorp/vagrant/issues/9897#issuecomment-394945635

**Comment by EnergeticPixels** [comment]
Closes #7

**Comment by EnergeticPixels** [comment]
closes #7
